Phonebook.Frontend/src/app/services/layout.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { OverlayContainer } from '@angular/cdk/overlay';
+import { Layout } from 'src/app/shared/models/enumerables/Layout';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LayoutService {
```

Comment:
You don't seem to leverage the class set in the body, so this is basically unused code?

Reply:
I use it in App.state.ts. I just tried removing it and that didn't go very well.

Reply:
Are using the CSS class you set here?

Reply:
Yes LayoutService gets used in App.state.ts and app.module.ts